app/Http/Controllers/PostController.php

```diff
@@ -64,19 +62,19 @@ public function show(Post $post)
 
     /**
      * Show the form for editing the specified resource.
-     *
      * @param  \App\Models\Post  $post
      * @return \Illuminate\Http\Response
+     * @return oldCategoryIds
```

Comment:
**remove** ` * @return oldCategoryIds`

`* @return \Illuminate\Http\Response` is OK